packages/edgeidentity/android/src/main/java/com/adobe/marketing/mobile/reactnative/edgeidentity/RCTAEPEdgeIdentityModule.java

```diff
@@ -42,13 +42,18 @@ public void extensionVersion(final Promise promise) {
     promise.resolve(Identity.extensionVersion());
   }
 
- @ReactMethod
-    public void getExperienceCloudId(final Promise promise) {
-        Identity.getExperienceCloudId(new AdobeCallback<String>() {
-            @Override
-            public void call(String s) {
+  @ReactMethod
+  public void getExperienceCloudId(final Promise promise) {
+      Identity.getExperienceCloudId(new AdobeCallbackWithError<String>() {
+        @Override
+          public void fail(AdobeError error) {
+                promise.reject("AdobeError", error.getErrorName());
```

Comment:
I think we can also provide an extension name when calling reject in Java code.  https://github.com/adobe/aepsdk-react-native/blob/36806b1a4add0a89f235d52d6e14335cd1d5d83a/packages/core/android/src/main/java/com/adobe/marketing/mobile/reactnative/RCTAEPCoreModule.java#L115

Reply:
Change to write it as this way right now
reject(EXTENSION_NAME, nil, error);
Sample output:
WARN  Possible Unhandled Promise Rejection (id: 0):
Error: The operation couldn’t be completed. (AEPCore.AEPError error 1.)

-----------------
If use 
reject(EXTENSION_NAME, FAILED_TO_CONVERT_EVENT_MESSAGE, error);

Sample output  would be generic
WARN  Possible Unhandled Promise Rejection (id: 0):
Error: Failed to get Experience Cloud ID

Let me know what you think